apps/closures-and-restrictions-map/src/components/Detail/DynamicDetail.tsx

```diff
@@ -48,6 +50,25 @@ export const DynamicDetail: React.FC<IDetail> = ({ featureProps, streetViewUrl }
 
     const { data: attachments } = useArcgisAttachments(DIGUPS_URL, objectId || 0);
 
+    const { data: disordersAttachments } = useArcgisAttachments(
+        DISORDERS_URL,
+        objectId || originalProperties?.objectid || 0,
```

Comment:
I know this is my ugly code, was just quickly testing at the time, but I think in case of disorders the `objectId` will never be valid and you thus need only

```suggestion
        originalProperties?.objectid || 0,
```

also if you can don't be afraid to get rid of the `|| 0`, that was also just a quick way to be typescript-valid at the time